index.html

```diff
@@ -0,0 +1,76 @@
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" 
+  integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+  <link rel="stylesheet" href="index.css">
+  <script type="text/javascript" src="https://moment.github.io/luxon/global/luxon.min.js "></script>
+  <title>Awesome-books</title>
+</head>
+<body>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container-fluid p-2">
+        <a class="navbar-brand" href="#">Awesome Books</a>
+        <div>
+            <ul class="d-flex align-items-center">
+                <li class="nav-item">
+                    <a class="nav-link desactive" href="#">List</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link desactive" href="#">Add new</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link desactive" href="#">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+    <div class="date">
+        <p id="render-date" class="fs-6 date position-absolute"></p>
+    </div>
+    
+    <section class="book-list" id="list-page">
+      <h1 class="book-list-title">All Awesome Books</h1>
+        <table id="book-list" class="table"></table>
+    </section>
+    
+    <section class="add-book-form" id="book-form-container">
+      <h2 class="add-new-book">Add a New Book</h2>
+        <form id="book-form">
+            <input id="title" type="text" placeholder="Title" required><br>
+            <input id="author" type="text" placeholder="Author" required><br>
+            <button class="btn" type="submit" id="form_btn">Add</button>
+        </form>
+    </section>
+    
+    <section id="contact" class="p-3 text-center d-none">
+        <h3 class="display-4 py-2 text-dark ">Contact Information</h3>
+        <p>If you have any question or you just want to say "Hello"? <br>
+            You can reach out to us!
+        </p>
+        <ul class="mt-4">
+            <li class="mb-2">
+                <h5>Ezra Yehuwalashet</h5>
+                <span>GitHub: <a class="text-dark" href="https://github.com/ezraermy">@ezraermy</a></span><br>
+                <span>LinkedIn: <a class="text-dark" href="https://www.linkedin.com/in/ezra-yehuwalashet">@Ezra Yehuwalashet</a></span>
+            </li>
+            <li class="mb-2">
+                <h5>Jose Soza</h5>
+                <span>GitHub: <a class="text-dark"  href="https://github.com/edgardsoza">@edgardsoza</a></span><br>
+                <span>LinkedIn: <a class="text-dark"  href="https://github.com/edgardsoza">@edgardsobalvarro</a></span>
+            </li>
+        </ul>
+    </section>
+    
+    <footer class="d-flex flex-column justify-content-center align-items-center p-3">
+        <span>Copyright &copy; 2023</span>
+        <span>Ezra Yehuwalashet & Jose Soza</span>
+    </footer>
```

Comment:
- [ ] **[OPTIONAL]** I think it would be a good idea to move the footer to the bottom of the page instead of leaving it hanging almost at the middle of the page.

  ![image](https://user-images.githubusercontent.com/81468675/221626437-39e39402-a376-40a0-9164-1550ff00f64b.png)